imports/ui/components/modals/modals.js

```diff
@@ -4,3 +4,23 @@ Template.main_modal.onCreated(function () {
   this.modalState = new ReactiveDict()
   this.modalState.set('type', this.data.type)
 })
+
+Template.modal_earn_pti.onCreated(function () {
+  this.stepState = new ReactiveDict()
+  this.stepState.set('currentStep', 1)
+})
+
+Template.modal_earn_pti.helpers({
+  currentStep () {
+    return `step-${Template.instance().stepState.get('currentStep')}`
+  }
+})
+
+Template.modal_earn_pti.events({
+  'click button.prev' () {
```

Comment:
This block of changes is also not going to be used/visible right now, but it allows the user to page through the instructions for earning more tokens